src/utils/phoenix-utils.js

```diff
@@ -1,15 +1,14 @@
-import queryString from "query-string";
 import uuid from "uuid/v4";
 import { Socket } from "phoenix";
 
 export function connectToReticulum() {
-  const qs = queryString.parse(location.search);
+  const qs = new URLSearchParams(location.search);
 
-  const socketProtocol = qs.phx_protocol || (document.location.protocol === "https:" ? "wss:" : "ws:");
+  const socketProtocol = qs.get("phx_protocol") || (document.location.protocol === "https:" ? "wss:" : "ws:");
   const [retHost, retPort] = (process.env.DEV_RETICULUM_SERVER || "").split(":");
   const isProd = process.env.NODE_ENV === "production";
-  const socketPort = qs.phx_port || (isProd ? document.location.port : retPort) || "443";
-  const socketHost = qs.phx_host || (isProd ? document.location.hostname : retHost) || "";
+  const socketPort = qs.get("phx_port") || (isProd ? document.location.port : retPort) || "443";
```

Comment:
I'm probably just misreading this, but should the ` || "43"` be ` document.location.port || "443"` for dev environments?

Reply:
Honestly, this code doesn't make any sense to me, because I don't understand why the websocket port we're using should be correlated with `document.location.port`. I may try to understand it and interrogate Greg about why it is the way that it is.